spark/src/test/java/org/opensearch/sql/spark/dispatcher/SparkQueryDispatcherTest.java

```diff
@@ -690,6 +695,7 @@ void testRefreshIndexQuery() {
     HashMap<String, String> tags = new HashMap<>();
     tags.put(DATASOURCE_TAG_KEY, "my_glue");
     tags.put(CLUSTER_NAME_TAG_KEY, TEST_CLUSTER_NAME);
+    tags.put(JOB_TYPE_TAG_KEY, JobType.BATCH.getText());
```

Comment:
@dai-chen @vamsi-amazon We don't support REFRESH XXX, means batch job right. it will not be long running job for now.

Reply:
yes, REFRESH only supports triggering manual refresh. Ref: https://github.com/opensearch-project/opensearch-spark/issues/100